src/schema/v2/me/mark_all_notifications_as_read_mutation.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLBoolean } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+
+export const markAllNotificationsAsReadMutation = mutationWithClientMutationId<
+  any,
+  any,
+  ResolverContext
+>({
+  name: "MarkAllNotificationsAsRead",
+  description: "Mark all unread notifications as read",
+  inputFields: {},
+  outputFields: {
+    success: {
+      type: GraphQLBoolean,
+      resolve: (result) => result.success,
+    },
+  },
+  mutateAndGetPayload: async (_, { updateNotificationsLoader }) => {
+    if (!updateNotificationsLoader) {
+      throw new Error("You need to be signed in to perform this action")
+    }
+
+    try {
+      await updateNotificationsLoader({ status: "read" })
+
+      return {
+        success: true,
+      }
+    } catch {
+      return {
+        success: false,
+      }
+    }
```

Comment:
or is it even better to use a common response format for mutations (for example, as in the screenshot), as is done for all mutations in Metaphysics?

![image](https://user-images.githubusercontent.com/3513494/190124214-93a8df5b-70ea-4cc9-afae-d25827d03972.png)